src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/SuggestionHandlers/FirstNameNodeSuggestionHandler.cs

```diff
@@ -71,10 +71,6 @@ internal override bool TryAddSuggestionsForNodeKind(IntellisenseData.Intellisens
 
                     intellisenseData.AddAdditionalSuggestionsForKeywordSymbols(curNode);
                 }
-                else if (IsBracketOpen(tok.Span.Lim, cursorPos, intellisenseData.Script))
```

Comment:
>IsBracketOpen [](http://example.com/codeflow?start=25&length=13)

fyi ... this will probably cause some failures in PAClient (I bet we have tests that need to be disabled) 
So we should buddy build and check.

Reply:
I agree. I will check that.

Reply:
A mitigation is to only do this is the flag is set.